### ISSUE TITLE
[URGENT] payment orders broken for the german market

### ISSUE THREAD

**Issue by luc-demeyer** [OCA/bank-payment#996]
I am opening this ticket since I see nobody reacting on  https://github.com/OCA/bank-payment/pull/993 whereas we have a serious issue on the german market hence this fix should be reviewed and merged asap.

## Module
account_sepa_pain_base

## Describe the bug
Germany banks are switching from pain.001.003.03 in Germany in favour of pain.001.001.03.
The address block generated by the OCA account_sepa_pain_base is not supported by the german banks resulting in leaving Odoo customer to switch to manual payments or switch to Odoo Enterprise account_sepa (or install my PR if they find it).

## To Reproduce
upload a payment order to the banks mentionned in https://github.com/OCA/bank-payment/pull/993 (and probably more to come in the coming weeks, Sparkasse is involved which is one of the largest German banks?
**Affected versions**:
All versions.

**Comment by rafaelbn** [comment]
Hello @OSevangelist , are you aware of this? Please could you help?

**Comment by rafaelbn** [comment]
Ping @tv-openbig @marylla @yweng8111 @chandni299

**Comment by tv-openbig** [comment]
@rafaelbn @luc-demeyer We can confirm this issue since a few days with another bank (Volksbank).  @fkantelberg is aware of this. I would propose to apply that fix on the customers database in order to let them check with their bank if that fixes the pain format issues. Thanks a lot to highlight this issue. Please contact me, if there is further activity required.

**Comment by fkantelberg** [comment]
@luc-demeyer Can you provide this patch for 14.0? In that case I can test it directly in 14.0

**Comment by luc-demeyer** [comment]
@fkantelberg 
cf. https://github.com/OCA/bank-payment/pull/998

**Comment by rafaelbn** [comment]
@luc-demeyer for a hurry up send an email to contributors@

**Comment by fkantelberg** [comment]
@luc-demeyer Could it be that the specification of `pain.001.001.03` changed? You can find new xsd files [here](https://www.ebics.de/de/datenformate/ergaenzende-dokumente) within the `SEPA_Schema_GBIC_3_ergaenzt_am_4.6.2021.zip` file. Inside you can see a changed xsd for pain.001.001.03 where they dropped PstCd and TwnNm. Would you consider updating the xsd files?

**Comment by luc-demeyer** [comment]
@fkantelberg 
I looked to the xsd files via the URL you gave : we see that the GBIC (German Banking Industry Committee) has published its own subset of the pain.001.001.03 protocol.
Seen the high impact of non-working pain files on the german market I propose that we proceed as follows:
1) merge the https://github.com/OCA/bank-payment/pull/998 and https://github.com/OCA/bank-payment/pull/993 Pull Requests asap. They have been proven to work in the meanwhile and Odoo Enterprise generates the same ,format hence we can feel comfortable that we do not break pain files for usage outside Germany.
2) keep this issue open for a little bit to capture the information from other countries. If other countries specify deviating pain.001.001.03 subsets than we have to add multiple xsd files to the module and create german, french, ... specific logic.

The fastest way forward would be to simply replace the current xsd files by the _GBIC_3 flavours but I am not sure this is the best way to move forward.
